Enh #18483: Add property `yii\log\Logger::dbEventNames` - an array wi… (#18484)
yiisoft/yii2: framework/CHANGELOG.md

2.0.41 under development
------------------------

> - Enh #18483: Add property `yii\log\Logger::dbEventNames` - an array with event names used to get statistical results (profiling) of DB queries (atiline)

Reviewer: ```suggestion
- Enh #18483: Add `yii\log\Logger::$dbEventNames` that allows specifying event names used to get statistical results (profiling) of DB queries (atiline)
```
